tests/integration/lexicon/test_daemon_link.py

```
@@ -0,0 +1,285 @@
+    async def test_the_probe_stops_when_the_cast_is_over(socket_path: Path):
+        link = DaemonLink(socket_path=socket_path, hello=_hello())
+        probing = asyncio.create_task(link.keep_attached(list, every=0.01))
+        await asyncio.sleep(0.02)
+
+        await link.close(status="ok")
+
+        await asyncio.wait_for(probing, timeout=1)
+        assert probing.done()
+
+    @staticmethod
+    async def test_a_reattach_replaces_what_the_daemon_held(socket_path: Path):
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_a_reattach_replaces_what_the_daemon_held`

Add return type annotation: `None`

(ANN205)